### DISCUSSION TITLE
Report unbudgeted amount in/above progress bar.

### DISCUSSION THREAD

**Discussion by tylerhcarter** [firefly-iii/firefly-iii#4691] [General]
**Description**
I would like to use Firefly to do a zero-based budget, where all my budgets minus my income equals zero. Currently I need to constantly use a calculator to identify just how much money is left to budget (for weird numbers especially).

**Solution**
List the remaining amount to budget above the progress bar on the budget page.

**What are alternatives?**
List the remaining amount at the bottom of the table.

**Comment by JC5** [comment]
Sorry, no.

https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/personal-finances/
